tests/Fixer/FunctionNotation/PhpdocToReturnTypeFixerTest.php

```diff
@@ -370,6 +370,36 @@ function bar() {}
             '<?php /** @return int */ fn(): int => 1;',
             '<?php /** @return int */ fn() => 1;',
         ];
+
+        yield 'arrow function (static)' => [
+            '<?php /** @return int */ static fn(): int => 1;',
+            '<?php /** @return int */ static fn() => 1;',
+        ];
+
+        yield 'support for arrow function' => [
```

Comment:
This one should be rather like:

```suggestion
        yield 'arrow function passed as an function argument' => [
```

Reply:
```suggestion
        yield 'arrow function (as argument ended with ,)' => [

```